playbooks/archive_artifacts.yml

```diff
@@ -0,0 +1,88 @@
+- hosts: "{{ target_hosts }}"
+  tasks:
+    - name: Create temporary directory to store artifacts
+      file:
+        path: "{{ artifacts_path }}"
+        state: directory
+      delegate_to: "localhost"
+      run_once: yes
+
+    - name: Grab host data
+      command: >
+               rsync
+               --archive
+               --compress
+               --verbose
+               --relative
+               --rsh 'ssh -o StrictHostKeyChecking=no'
+               --ignore-missing-args
+               {{ inventory_hostname }}:{{ item }}
+               {{ artifacts_path }}/{{ ansible_hostname }}
+      with_items:
+        - "/openstack/log/{{ ansible_hostname }}-lxc"
```

Comment:
Previously we were grabbing `/openstack/log/$HOSTNAME-*`, if we do `/openstack/log/{{ ansible_hostname }}-lxc` here we miss out on a bunch of logs and also the LXC logs should already be included in `/var/log` I believe?

Here's all the logs for services running on the host itself:

```
root@liberty-aio:/openstack/log# ls -dl /openstack/log/$HOSTNAME-*
drwxr-xr-x 2 cinder  cinder  4096 Aug 23 22:15 /openstack/log/liberty-aio-cinder
drwxr-xr-x 2 syslog  adm     4096 Aug 23 21:11 /openstack/log/liberty-aio-haproxy
drwxr-xr-x 2 neutron neutron 4096 Aug 23 22:26 /openstack/log/liberty-aio-neutron
drwxr-xr-x 2 nova    nova    4096 Aug 23 22:22 /openstack/log/liberty-aio-nova
drwxr-xr-x 2 syslog  syslog  4096 Aug 23 22:41 /openstack/log/liberty-aio-swift
root@liberty-aio:/openstack/log#
```

Reply:
Agreed.

Reply:
I believe that should be taken care of by line 68.

Reply:
Line 68 only grabs container data, what if you're on an AIO that has cinder, haproxy, neutron, nova, and swift running outside of containers?  You'd want to grab that data which was done previously using $HOSTNAME-* (this doesn't match containers, since that'd be $HOSTNAME_*).